### PR TITLE
Zj-cp/add performance tasks

### PR DESCRIPTION
**Pull request by zacjones93**
@Creeland and I worked through the course making performance tasks that reflect the outcomes of the course

Course Outcomes:
- Bulk insert and export from csv
- Casting types in SQL
- Define Custom Types (like ENUMs) for data integrity
- Analyze query performance for optimal queries
- Write concise queries with CTE's (Common Table Expressions) and do/declare blocks
- Filter Aggregate data with having
- Guarantee database interactions with Transactions
- Complex pattern matching

### CODE REVIEW

File: exercises/06-filter-data.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Filter Data
+
+## 01.
+Return all the users that are female and in their 20's that have purchased more than 100 products.
```
Comment (Creeland) [inline exercises/06-filter-data.md:4]: We can probably reword this to "Your goal is to return all female users in their 20's that have purchased more than 100 products. You will need to filter aggregate and row data."

File: exercises/solutions/01-import-data.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Import Data
+
+### Create Users
+```sql
+copy users(user_handle,first_name,last_name,email,gender,age,language) FROM '/Users/zac/Documents/01. Projects/advanced-sql-pt/user-data.csv' DELIMITER ',' CSV HEADER;
```
Comment (Creeland) [inline exercises/solutions/01-import-data.md:5]: We could make the paths generic. `/Users/{YOUR_USER}/{PATH_TO_PROJECT}/user-data.csv`

File: exercises/solutions/07-transactions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# 07
+
+## 01.a
```
Comment (Creeland) [inline exercises/solutions/07-transactions.md:3]:
```suggestion
## 01
```

File: exercises/solutions/01-import-data.md (hunk 2 above)
Comment (zacjones93) [reply exercises/solutions/01-import-data.md:5]: good catch

**Comment by zacjones93** [comment]
@all-contributors-action please add @Creeland  for review, content
